frontend/encore/simple-example.rst

```diff
@@ -163,7 +121,7 @@ Great! Use ``require()`` to import ``jquery`` and ``greet.js``:
     // assets/js/app.js
 
     // loads the jquery package from node_modules
-    const $ = require('jquery');
+    var $ = require('jquery');
```

Comment:
I'm not sure using `var` should be encouraged nowadays since it's a lot less intuitive than `const`/`let`.
Did you change it to be more consistent with the rest of the doc? Wouldn't it be better to go over the others `var` instead?